packages/cli/package.json

```diff
@@ -25,19 +27,21 @@
   "devDependencies": {
     "@types/configstore": "^5.0.1",
     "@types/inquirer": "^8.2.0",
-    "@types/ws": "^8.5.3",
+    "@types/jest": "^29.5.12",
     "@types/mocha": "10.0.2",
+    "@types/ws": "^8.5.3",
+    "jest": "^29.7.0",
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Use the more performant and modern `vitest` instead, which also ships with types.‏